src/main/java/uk/gov/gchq/palisade/service/PolicyConfiguration.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service;
+
+import uk.gov.gchq.palisade.resource.Resource;
+
+import java.util.List;
+
+public interface PolicyConfiguration {
+
+    List<? extends PolicyCacheWarmerFactory> getPolicies();
```

Comment:
Can we have some JavaDoc to understand the purpose of these methods and therefore what is the expected functionality of these methods

Reply:
Added JavaDoc